Controller/ChangePasswordController.php

```diff
@@ -86,6 +86,15 @@ public function changePasswordAction(Request $request)
 
         return $this->render('@FOSUser/ChangePassword/change_password.html.twig', array(
             'form' => $form->createView(),
+            'data' => $this->setUserData(),
         ));
     }
+
+    /*
+     * Function to override to send data to the template
```

Comment:
Yes, I don't have the whole php wording yet ;-) I have learned Symfony (+ Git, Github, etc.) by myself (with tutorials) but it's better to use the good words. Thanks